client/v2/autocli/common.go

```diff
@@ -119,7 +119,7 @@ func (b *Builder) outOrStdoutFormat(cmd *cobra.Command, out []byte) error {
 			return err
 		}
 	}
-	_, err = fmt.Fprintln(cmd.OutOrStdout(), string(out))
+	fmt.Fprintln(cmd.OutOrStdout(), string(out))
 
 	return nil
```

Comment:
Possibly we should keep storing the err, but return it here.
If it is nil then it'll return nil anyway.

Reply:
really great point!